src/test/scala/cognite/spark/v1/OAuth2Test.scala

```diff
@@ -49,8 +65,7 @@ class OAuth2Test
       assert(false) // fail if it works
     }
     catch {
-      case e: InvalidAuthentication => assert(true)
-      case _ => assert(false)
+      case e => assert(true)
```

Comment:
Why not use the the `intercept [Exception]`? Or you can use sparkIntercept if it throws SparkException

Reply:
how do we do that exactly :P it was me being a noob about scala test utilities hehe

Reply:
fixed, thanks!!

Reply:
Cool! Don't be afraid to ask how to use these things :)